Apply copy editor's suggestions

Nashaath opened the pull request:
## Purpose
Apply copy editor's suggestions on `Ballerina Expression Syntax`.

anupama-pathirage commented on en/docs/apis/create-and-publish-your-first-api.md:
```diff
@@ -3,14 +3,14 @@
     
     !!!note
-        In this quick start guide, let's use the mock service [http://www.mocky.io/v2/5185415ba171ea3a00704eed](http://www.mocky.io/v2/5185415ba171ea3a00704eed) that is made available by [https://www.mocky.io/](https://www.mocky.io/). Click the mock service URL to test the service. You can see the following JSON message:`{"hello": "world"}`
+        In this quick start guide, we use the mock service at [http://www.mocky.io/v2/5185415ba171ea3a00704eed](http://www.mocky.io/v2/5185415ba171ea3a00704eed), which is made available by [https://www.mocky.io/](https://www.mocky.io/). Click the mock service URL to test the service. You'll see the following JSON message:**{"hello": "world"}**.
```
```suggestion
        In this quick start guide, we use the mock service at [http://www.mocky.io/v2/5185415ba171ea3a00704eed](http://www.mocky.io/v2/5185415ba171ea3a00704eed), which is made available by [https://www.mocky.io/](https://www.mocky.io/). Click the mock service URL to test the service. You'll see the following JSON message: **{"hello": "world"}**.
```

anupama-pathirage replied:
Instead of bold , use back ticks with syntax highlightign as json. i.e. ```json {"hello": "world"}```

anupama-pathirage commented on en/docs/apis/create-and-publish-your-first-api.md:
```diff
@@ -21,60 +21,59 @@ Follow the procedure below to create an API using the mock REST service you test
+    2. Enter **hello** as the URI pattern.
     3. Click the **+** icon to add the resource.
-    4. Click **Save**. You can see a confirmation message to proceed with the changes.
+    4. Click **Save**. You'll see a confirmation message to proceed with the changes.
```
Is the contraction `'ll` appropriate for formal writing?

Nashaath replied:
We have avoid this approach generally but since the copy editor had suggested this throughout most content, I checked a few microsoft style-based docs and found similar usage [1]. Checked with the Choreo writers and we thought of accepting the suggestion. 
[1] https://docs.microsoft.com/en-us/microsoftteams/troubleshoot/meetings/how-to-join-a-teams-meeting#join-with-a-computer-or-laptop
We can discuss all such concerns and finalize guidelines on how we wan't to proceed.

anupama-pathirage commented on en/docs/apis/create-and-publish-your-first-api.md:
```diff
@@ -21,60 +21,59 @@ Follow the procedure below to create an API using the mock REST service you test
+      ![](../assets/img/apis/response.png)
 
-Now you are ready to publish the API to the Choreo Devportal so that external consumers can consume the API.
+Now you are ready to publish the API to the Choreo Develope Portal so that external consumers can consume the API.
```
```suggestion
Now you are ready to publish the API to the Choreo Developer Portal so that external consumers can consume the API.
```

anupama-pathirage commented on en/docs/references/ballerina-expression-syntax.md:
```diff
@@ -71,348 +71,348 @@ The boolean data type has one of the two possible values, `true` or `false`.
+### Checking the equality of basic types 
 
-In the [Ballerina language](https://ballerina.io/), basic types are `string`, `int`, `float`, `decimal`, `boolean`, and `nil`. You can compare the values of these types using the `==` binary operator. The values can come from a literal or a variable. Similarly `!=` operator is used to check the inequality. Note that the resulting value from these operators is always of `boolean` type. 
+In the [Ballerina language](https://ballerina.io/), basic types are **string**, **int**, **float**, **decimal**, **boolean**, and **nil**. You can compare the values of these types using the **==** binary operator. The values can come from a literal or from a variable. Similarly the **!=** operator is used to check the inequality. Note that the resulting value from these operators is always of `b ...
```
```suggestion
In the [Ballerina language](https://ballerina.io/), basic types are **string**, **int**, **float**, **decimal**, **boolean**, and **nil**. You can compare the values of these types using the **==** binary operator. The values can come from a literal or from a variable. Similarly, the **!=** operator is used to check the inequality. Note that the resulting value from these operators is always of `boolean` type. 
```

anupama-pathirage commented on en/docs/references/ballerina-expression-syntax.md:
````diff
@@ -71,348 +71,348 @@ The boolean data type has one of the two possible values, `true` or `false`.
   ```
   
-- Concatenating a string literal and a variable (`name`).
+- Concatenates a string literal and a variable (**name**):
````
I don't agree to have bold for code variables and syntaxes. IMO, we should use backticks for them and use bold only for UI elements and texts.

anupama-pathirage commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,176 +1,175 @@
+4. Under **Create with Choreo**, enter **covid-stats** as the name of your service.
 
-5. Select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
+5. Select **GET** as the HTTP method and enter **activecases** as the relative path from host.
```
```suggestion
5. Select **GET** as the HTTP method and enter **activecases** as the relative path from the host.
```

anupama-pathirage commented on en/docs/references/ballerina-expression-syntax.md:
```diff
@@ -36,9 +36,9 @@ A signed integer is an integer with the  `+` or `-` character in the front, indi
+### Decimal floating-point number
 
-A decimal floating-point number is a number with a decimal point. You have the option to use `+` or `-` characters in the front to indicate the sign.
+A decimal floating-point number is a number with a decimal point. You can precede the numbers with the + or - characters to indicate the sign.
```
`+ or -` is not highlighted in this case. But it is in the `Signed integer` topic. We should follow the same convention everywhere.  As per the comment https://github.com/wso2/docs-choreo-dev/pull/90/files#r636775634 I feel the backticks are the best option here.

Nashaath replied:
Since this is about decimal numbers, if we format + and - in bold it will get unnecessarily highlighted more than the decimal aspect. Whereas in Signed integer, we have to highlight + and - . 
Agree that backticks are the best option here.

anupama-pathirage commented on en/docs/references/ballerina-expression-syntax.md:
```diff
@@ -71,348 +71,348 @@ The boolean data type has one of the two possible values, `true` or `false`.
+### Numeric multiplication
 
-You can use the `*` operator for number multiplication. The number values can come from a literal or a variable.
+You can use the ***** (asterisk) operator for numeric multiplication. The number values can come from a literal or from a variable.
```
`*****` is not rendered properly . Lets use back ticks.

Nashaath replied:
+1

anupama-pathirage commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,176 +1,175 @@
+Click **API Calls** and then complete the following steps to configure the API trigger to use the HTTP connection that connects the public API used in this scenario to it.
 
-    1. Click **HTTP**.
+1. Click **HTTP**.
```
These numbers are not properly indented and sub numbers are not rendered. Check all the other places as well.

![image](https://user-images.githubusercontent.com/16300038/119116807-3e433100-ba46-11eb-8c1d-476010bcbe87.png)

anupama-pathirage replied:
It seems we are starting from number one again under the new section. But it is really confusing as the text in between is not appear as a separator for the sections.

anupama-pathirage replied:
I think we better have sub steps as sub topics with proper topic under the main sub topic of `Step 1: Create the service`

Nashaath replied:
Yeah. This is the numbering related concern I mentioned. It is confusing. So need to see whether we can be flexible with the number of steps under a procedure. Choreo doc team members felt that it is ok to have 10 steps or so under a section.

Nashaath commented on en/docs/apis/create-and-publish-your-first-api.md:
```diff
@@ -21,60 +21,59 @@ Follow the procedure below to create an API using the mock REST service you test
-    - **Endpoint**: http://www.mocky.io/v2/5185415ba171ea3a00704eed
+    - **API Name**: **HelloWorld**
+    - **API Version**: **1.0.0**
+    - **Endpoint**: **http://www.mocky.io/v2/5185415ba171ea3a00704eed**
```
Agree. It is confusing when the field names and input values are formatted in the same way.

anupama-pathirage commented on en/docs/references/ballerina-expression-syntax.md:
```diff
@@ -1,12 +1,12 @@
 # Ballerina Expression Syntax
```
Shouldn't we change this to `Ballerina expression syntax`?

Nashaath replied:
We use title-style capitalization for the page title and sentence-style capitalization for all other subsection headings. Since this is the page title it should be in title-style capitalization.

Nashaath commented:
Updated to format user input values, variables, and function/expression syntax with backticks.

Also changed the numbering to follow a more flexible approach.

anupama-pathirage commented on en/docs/references/ballerina-expression-syntax.md:
```diff
@@ -71,237 +71,237 @@ The boolean data type has one of the two possible values, `true` or `false`.
 #### Example
 
-- Converting the value of the integer variable (`total`) to a string value.
+- This converts the value of an integer variable (`total`) to a string:
```
In other places, we have newline before the example. Shouldn't we add new line here too?

Nashaath replied:
Yes, we need. 
Added a new line.

anupama-pathirage commented on en/docs/references/ballerina-expression-syntax.md:
```diff
@@ -312,111 +312,110 @@ The following operators are available in the expression editor for numeric value
 ### Checking whether a header is present in the request
 
-Applications sometimes need to check the availability of a header before proceeding to something like reading the header value. You can use the  `hasHeader`  function in the `request` object for this purpose.
+Applications sometimes need to check the availability of a header before proceeding to something like reading the header value. You can use the `hasHeader`  function in the request object for this purpose.
```
Why are we removing backtick from `request`?

Nashaath replied:
Removed because in such occurrences `request` is not really the variable name. Does it make sense to use backticks in such occurences?

Nashaath replied:
Formatted with backticks.

RukshiW replied:
There is an extra space after `has header`?

anupama-pathirage commented on en/docs/references/ballerina-expression-syntax.md:
```diff
@@ -312,111 +312,110 @@ The following operators are available in the expression editor for numeric value
+### Reading the JSON payload of the request
 
-JSON is a common content type used for HTTP communication. The `request` variable’s `getJsonPayload` function can be used to read the JSON payload sent with the HTTP request. Note that the `getJsonPayload` function’s return type is `json|ClientError`.
+JSON is a common content type used for HTTP communication. You can use the request variable’s `getJsonPayload` function to read the JSON payload sent with the HTTP request. Note that the `getJsonPayload` function’s return type is `json|ClientError`.
```
Why are we removing backtick from `request`?

anupama-pathirage replied:
Check all the places.

Nashaath replied:
Removed because in such occurrences `request` is not really the variable name. Does it make sense to use backticks in such occurrences?

Nashaath replied:
Formatted appropriate occurrences of the term request with backticks.

anupama-pathirage commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
 Now you have completed designing your service. It looks as follows.
 
-![No Code View](../assets/img/services/choreo-service-low-code-view.png)
+- In the low-code diagram
```
Shouldn't we say `In the low-code view`?

Nashaath replied:
+1
Updated.

shaniR commented on en/docs/apis/create-and-publish-your-first-api.md:
```diff
@@ -3,43 +3,41 @@
-1. Access the Choreo Console via [https://console.choreo.dev/](https://console.choreo.dev/).
-2. Sign in using either your GitHub or Google credentials.
-3. Go to the **APIs** card and click **Get Started**. This takes you to the **Create API** page. 
+1. Access the Choreo Console via https://console.choreo.dev/ and sign in using your Google or GitHub credentials.
```
Shall we consider re-phrasing this to "Sign in to the Choreo Console at via https://console.choreo.dev/"?

Nashaath replied:
+1

RukshiW replied:
Having a link at the end of a sentence is recommended to make the documentation more usable (since it is easier and quicker to click a link to the right).

Nashaath replied:
Applied suggestion in https://github.com/wso2/docs-choreo-dev/pull/108

shaniR commented on en/docs/apis/create-and-publish-your-first-api.md:
```diff
@@ -3,43 +3,41 @@
 Choreo’s API management capabilities allow you to create, publish, and manage all aspects of an API and its lifecycle.
 This quick start guide walks you through the steps to quickly create and publish an API using Choreo. 
```
Should we explicitly mention that we are creating a REST API?

Nashaath replied:
+1

shaniR commented on en/docs/apis/create-and-publish-your-first-api.md:
```diff
@@ -3,43 +3,41 @@
 
 ## Step 2: Deploy the API
-Follow the procedure below to create a revision of the API and deploy it:
+Follow this procedure to create a revision of the API and deploy it:
```
Should we mention why we are deploying it? "so that the API can be invoked" ? and also, where the API gets deployed by default?

Nashaath replied:
Changed to "Follow this procedure to create a revision of the API and deploy it so that you can try it out:"
Did not mention where the API gets deployed by default because it can be too much information at this point.

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
 # Create Your First Service
 
-To learn how to create a service, let's try a simple scenario. In this example, you are creating a service that consumes a public API that fetches the current active COVID-19 cases in a specified country, testing it, deploying it, and then observing its performance.
+To learn how to create a service, let's look at a simple scenario. In this example, you'll create a service that consumes a public API that fetches the current active COVID-19 cases in the United States. Then you'll test it, deploy it, and observe its performance.
```
Shouldn't we add a similar description for the create and publish an API page?

Nashaath replied:
In the `Create and Publish Your First API` page we have a prerequisites section that describes the mock rest service used in the quick start. If we are to have a similar section we might have to consider removing that and having a similar description. Else there can be unnecessary repetition. WDYT?

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
+To create the service, follow this procedure: 
 
-1. Access the Choreo Console via https://console.choreo.dev/.
+1. Access the Choreo Console via https://console.choreo.dev/ and sign in using your Google or GitHub credentials.
```
Shall we consider re-phrasing this to "Sign in to the Choreo Console at via https://console.choreo.dev/"?

Nashaath replied:
+1

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
-2. Sign in using either your Google or GitHub credentials.
-    
-3. In the **Services** page, click **Create**. You are directed to the low-code view.
+2. On the **Services** page, click **Create**. You are directed to the **Create Service** page.
```
Shouldn't this be "You'll be directed" ?

Nashaath replied:
Let's finalize the approach to take on this and then update all places appropriately.

Nashaath replied:
Since we decided to limit future-tense to tutorial introductions and so on, I did not apply the suggestion to procedure steps.

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
+2. On the **Services** page, click **Create**. You are directed to the **Create Service** page.
 
-4. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
+3. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
```
shouldn't the user input be in bold? https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions

shaniR replied:
The format used in the create api page is a little different. For e.g. in that we have mentioned the field in bold, and then the value. Should we follow a consistent way of presenting the inputs?

Nashaath replied:
Although the copy editor suggested bold to format user input values, we want to suggest the use of backticks because it can be confusing when the field names and user input values are formatted in the same way.

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
+3. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
 
-5. Select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
+4. Select **GET** as the HTTP method and enter `activecases` as the relative path from the host.
```
shouldn't the user input be in bold? https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions

Nashaath replied:
Although the copy editor suggested bold to format user input values, we want to suggest the use of backticks because it can be confusing when the field names and user input values are formatted in the same way.

shaniR commented on en/docs/apis/create-and-publish-your-first-api.md:
```diff
@@ -3,43 +3,41 @@
-5. In the **Create API from REST API** form, fill in the details as follows:
+3. On the **Create API** page, go to the **Start from scratch card** and click **Next**.
+4. In the **Create API from REST API** form, fill in the details as follows:
     - **API Name**: `HelloWorld`
```
shouldn't the user input be in bold? https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions

Nashaath replied:
Although the copy editor suggested bold to format user input values, we want to suggest the use of backticks because it can be confusing when the field names and user input values are formatted in the same way.

RukshiW replied:
+1 to have the user input values formatted with black ticks for the same reasons given by Nashaath.

shaniR commented on en/docs/apis/create-and-publish-your-first-api.md:
```diff
@@ -3,43 +3,41 @@
+3. On the **Create API** page, go to the **Start from scratch card** and click **Next**.
+4. In the **Create API from REST API** form, fill in the details as follows:
     - **API Name**: `HelloWorld`
     - **API Version**: `1.0.0`
```
here too

Nashaath replied:
Although the copy editor suggested bold to format user input values, we want to suggest the use of backticks because it can be confusing when the field names and user input values are formatted in the same way.

shaniR commented on en/docs/apis/create-and-publish-your-first-api.md:
```diff
@@ -3,43 +3,41 @@
+4. In the **Create API from REST API** form, fill in the details as follows:
     - **API Name**: `HelloWorld`
     - **API Version**: `1.0.0`
-    - **Endpoint**: http://www.mocky.io/v2/5185415ba171ea3a00704eed
```
here too

Nashaath replied:
Although the copy editor suggested bold to format user input values, we want to suggest the use of backticks because it can be confusing when the field names and user input values are formatted in the same way.

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
-6. Click **API Calls**, and then enter information as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
+     ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
+
+5. Click **Save API**. Now you have saved the API trigger that starts your service.
```
What does an "API trigger" mean? It's a new term that is used in this document.

Nashaath replied:
@RukshiW, any specific reason to use the term API trigger?

Nashaath replied:
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
+
+5. Click **Save API**. Now you have saved the API trigger that starts your service.
+
+6. Click **API Calls** and then complete the following steps to configure the API trigger to use the HTTP connection that connects the public API used in this scenario to it:
```
Should we make this a sub section? Configure the service maybe?

shaniR replied:
I feel the sentence is too long. Can we re-phrase it if we are not going to make it a sub section?

Nashaath replied:
+1 to have a subsection. But need to go through the complete flow and see the best way to do this.

Nashaath replied:
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
-        Click **Save**.
-        
+         
+    2. Enter the following information and then click **Save**:
```
The way we present input value, field vs value is different from place to place. shall we try to follow one approach?

Nashaath replied:
The plan is to suggest the use of backticks for user input values and bold formatting for field names.

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
         ![Update Existing Connection](../assets/img/services/update-existing-connection.png)
         
-    4. Click **API Calls**, and then click on the existing connection.
+    4. Click **API Calls** and then click the existing connection.
```
We can refer to the existing connection by the connection name we added above, can't we?

Nashaath replied:
Since the UI shows the added connection as an existing connection, the current sentence seems ok. 
<img width="894" alt="Screenshot 2021-05-28 at 10 12 49" src="https://user-images.githubusercontent.com/9528121/119931028-3b909080-bf9e-11eb-8cb1-56f980d2a9bd.png">

I feel the sentence might become longer if we try to mention the name. WDYT?

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
+    6. Click **Save & Done**.
     
-7. To cast all the data retrieved via the API connected to the service before further processing, add a new custom statement as follows:
+       Now your API trigger is connected to the public API that fetches the active COVID-19 cases in the selected country.
```
What does it mean or the API trigger to be connected?

Nashaath replied:
Might be better to rephrase this depending on how we change the term "API trigger". 
@RukshiW any suggestions on this?  
Can we just say "Now you have configured the service to fetch the active COVID-19 cases in the selected country"?

RukshiW replied:
+1 for "Now you have configured the service to fetch the active COVID-19 cases in the selected country".

Nashaath replied:
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
-7. To cast all the data retrieved via the API connected to the service before further processing, add a new custom statement as follows:
+       Now your API trigger is connected to the public API that fetches the active COVID-19 cases in the selected country.
+
+7. Add a new custom statement as follows to cast all the data retrieved via the API connected to the service before further processing:
```
what we are trying to do here, and why, is not very clear.

Nashaath replied:
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
     3. In the **Iterable Expression** field, enter `jsonArray`. Then click **Save**.
     
-    4. To apply the filter mentioned, add a custom statement within the ForEach statement. To do this, click the **+** icon just below the last ForEach statement you added.
+    4. To apply the filter, add a custom statement within the ForEach statement. To do this, click the **+** icon just below the last ForEach statement you added.
```
What's the filter?  filter the dates where the active cases have exceeded 5,000?

shaniR replied:
Do we need to mention it again? I feel it confuses the reader. If we continue with steps I think it would be better, because this is mentioned above.

Nashaath replied:
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
-    ![Add Respond Statement](../assets/img/services/add-respond-statement.png)
-    
-    In the **Respond Expression** field, enter `response`, and then click **Save**.
+9. Add a Respond statement as follows so that the result of the above processing is returned as a response:
```
why do we add a colon here? because we are introducing a list?

Nashaath replied:
Yes.

Nashaath replied:
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108 
The plan is to go with the recommendation in https://developers.google.com/style/procedures#introductory-sentences with regard to the use of a colon when a sentence immediately precedes the procedure.

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
 ## Step 2: Test the service
 
-To test the `covid-stats` service you created, follow the procedure below:
+To test the `covid-stats` service you created, follow this procedure:
```
shouldn't this be bold?

Nashaath replied:
We are using bold for field names and other occurrences of UI strings.

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
 3. Click **Try it out**, and then click **Execute**.
 
-The search results for COVID-19 statistics of the United States are displayed as the server response as shown below.
+The following figure shows the server response to the search results for the United States COVID-19 statistics:
```
Why do we use the colon here?

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
 ## Step 3: Deploy the service
 
-To deploy the `covid-stats` service, follow the procedure below:
+To deploy the `covid-stats` service, follow this procedure:
```
bold?

Nashaath replied:
Guideline is to go with backtick.

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
 2. To deploy the service, click **Deploy**.
 
-    The status of the service changes to **Deployed**, and the following logs appear to indicate that the product is successfully deployed.    
+    The status of the service changes to **Deployed**, and the following logs show that the product is successfully deployed:    
```
Why do we use bold text here?

Nashaath replied:
Guideline is to go with bold for UI elements.

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
+      ![Deployment Logs](../assets/img/services/deployment-logs.png)
 
-3. Once the service is deployed, click on the **Go Live** tab below the low-code design view. Then click **://cURL** and copy the cURL command that is displayed.
+3. Once the service is deployed, click the **Go Live** tab below the low-code diagram. Then click **://cURL** and copy the cURL command that is displayed.
```
Shall we re-phrase to " Then click **://cURL** and copy the cURL command"

Nashaath replied:
+1
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108.

shaniR commented on en/docs/services/create-your-first-service.md:
```diff
@@ -1,168 +1,161 @@
+      ![covid-stats-log](../assets/img/services/covid-stats-log.png)
 
-    In the Choreo Development Console, **Go Live** tab, the requests are logged as follows.
+5. In the Choreo Development Console **Go Live** tab, the requests are logged as follows:
```
What is the Choreo  Development Console?

Nashaath replied:
Changed to Choreo Console.

Nashaath commented on en/docs/apis/create-and-publish-your-first-api.md:
```diff
@@ -3,43 +3,41 @@
  
 ## Step 1: Create an API from scratch
-Follow the procedure below to create an API using the mock REST service you tested in the prerequisites section:
+Follow this procedure to create an API using the mock REST service you tested in the prerequisites section:
```
Since this is just below the prerequisites section do we need to consider linking back?

Nashaath commented:
The change suggestions given here have been addressed via [1] because there were too many conflicts that couldn't be fixed manually and pushed to `Nashaath-patch-2`.
